Mathlib/RingTheory/Bialgebra/Hom.lean

```diff
@@ -160,6 +160,7 @@ theorem coe_toCoalgHom (f : A →ₐc[R] B) : ⇑(f : A →ₗc[R] B) = f :=
   rfl
 
 lemma toCoalgHom_apply (f : A →ₐc[R] B) (a : A) : f.toCoalgHom a = f a := rfl
+lemma toLinearMap_apply (f : A →ₐc[R] B) (x : A) : f.toLinearMap x = f x := rfl
```

Comment:
This ordering is a little weird, I'd expect either the grouping to be coe vs apply or CoalgHom vs LinearMap, this is neither.